src/Synapse/Synapse.Test/ScenarioTests/SqlPoolBackupTests.ps1

```diff
@@ -105,6 +105,72 @@ function Test-SqlPoolRestorePoint
 	}
 }
 
+function Test-SqlPoolGeoBackup
+{
+	# Setup
+	$testSuffix = 'ps2502'
+	$params = Get-SqlPoolBackupTestEnvironmentParameters $testSuffix
+
+	try
+	{	
+	    $SqlPoolGeoBackupGet =Get-AzSynapseSqlPoolGeoBackup -ResourceGroupName $params.rgname -WorkspaceName $params.workspaceName
+        Assert-AreEqual $params.sqlPoolName $SqlPoolGeoBackupGet[0].Name
+        Assert-Null $SqlPoolGeoBackupGet[0].ElasticPoolName
+        Assert-NotNull $SqlPoolGeoBackupGet[0].Edition
+        Assert-NotNull $SqlPoolGeoBackupGet[0].LastAvailableBackupDate
+
+        $SqlPoolGeoBackupGetByPool =Get-AzSynapseSqlPoolGeoBackup -ResourceGroupName $params.rgname -WorkspaceName $params.workspaceName -Name $params.sqlPoolName
+        Assert-AreEqual $params.sqlPoolName $SqlPoolGeoBackupGetByPool.Name
+        Assert-Null $SqlPoolGeoBackupGetByPool.ElasticPoolName
+        Assert-NotNull $SqlPoolGeoBackupGetByPool.Edition
+        Assert-NotNull $SqlPoolGeoBackupGetByPool.LastAvailableBackupDate
+    }
+	finally
+	{
+		# Cleanup
+		Remove-SqlPoolBackupTestEnvironment $testSuffix
+	}
+}
+
+function Test-DroppedSqlPool
+{
+	# Setup
+	$testSuffix = getAssetName
+	Create-SqlPoolBackupTestEnvironment $testSuffix
+    $params = Get-SqlPoolBackupTestEnvironmentParameters $testSuffix
+
+	try
+	{	
+        Remove-AzSynapseSqlPool -ResourceGroupName $params.rgname -WorkspaceName $params.workspaceName -Name $params.sqlPoolName -Force
+
+        Wait-Seconds 300
```

Comment:
>Wait-Seconds 300 [](start = 8, length = 16)

Just curious, does this command take effect in playback too? If so, better to check if the current mode is Record or not. (Not mandatory)